docs/getting_started.md

```diff
@@ -1,65 +1,54 @@
-# <a name="getting-started">Getting started</a>
+# Getting started
 
-1. [Getting started](#getting-started)
-    1. [Prerequisites](#prerequisites)
-    2. [Installation](#installation)
-    3. [CMake build system](#cmake)
-    4. [Build Acts on lxplus](#build-lxplus)
-    5. [Build Acts on your local machine](#build-local)
-2. [Using Acts in your own cmake project](#using-acts)
-3. [Documentation](#documentation)
+## Prerequisites
 
-## <a name="prerequisites">Prerequisites</a>
+The following dependencies are required to build the Acts Core library:
 
-Only few dependencies are required to build the Core library of Acts. A list of
-prerequisites required is given below with the version numbers indicating which
-versions were tested. Older versions may or may not work, feedback is very
-welcome.
-
-The following dependencies are required:
-
-*   A C++14 compatible compiler, e.g. [gcc](https://gcc.gnu.org) (>= 6.2) or [clang](http://clang.llvm.org) (>= 4.0)
-*   [cmake](https://cmake.org) (>= 3.7)
-*   [boost](http://boost.org) (>= 1.62, with `program_options` and `unit_test_framework`)
+*   A C++17 compatible compiler (recent versions of either gcc and clang should work)
+*   [CMake](https://cmake.org) (>= 3.11)
+*   [Boost](http://boost.org) (>= 1.69, with `program_options` and `unit_test_framework`)
```

Comment:
`filesystem` for the examples, I think it isn't listed below. Should be mentioned somewhere.

Reply:
I added `filesystem` to the list.